app/services/idv/steps/in_person/state_id_step.rb

```diff
@@ -38,6 +38,11 @@ def call
           if flow_session['Idv::Steps::InPerson::AddressStep']
             redirect_to idv_in_person_verify_info_url
           end
```

Comment:
Since this is being checked here, and also for the 50/50 state, you'll want to set this property in update, when you add that back in. You could consider adding a check for pii_from_user[:address1] here, and then after that's deployed, you could delete the AddressStep check.

Also above on line 28, you'll want to change `mark_step_complete(:address)` to `flow_session['Idv::Steps::InPerson::AddressStep'] = true` or this will cause 500s in the 50/50 state when you deploy your delete PR. Same for line 35, setting it to false.

Reply:
Yes, it's fine to deploy as-is for now. You'll just want to make sure you've updated those and deployed before you deploy a delete PR.